src/jit/gentree.cpp

```diff
@@ -6150,32 +6170,37 @@ GenTreePtr Compiler::gtNewStringLiteralNode(InfoAccessType iat, void* pValue)
 
     switch (iat)
     {
-        case IAT_VALUE: // The info value is directly available
```

Comment:
Does the IAT_VALUE case never exist?

Reply:
Yes, it never exists and can't exist.  A string is a GC ref and must live in the GC heap. You can't embed  a fixed address of a GC ref.